LivrETS/Scripts/manageusers.js

```diff
@@ -16,14 +16,116 @@ You should have received a copy of the GNU General Public License
 along with this program.  If not, see <http://www.gnu.org/licenses/>.
 */
 $(document).ready(function () {
-    $("thead>tr").find("th").addClass("text-center");
+    //Id for current user
+    var currentId;
+    var listRoles;
+
+    //init manage users table width datables plugins
+    
+    $('table').DataTable({
+        processing: true,
+        ajax: {
+            url: "/Admin/ListUsers",
+            type: "POST",
+            dataType: "JSON",
+            dataSrc: function (val) {
+                currentId = val.current_id;
+                listRoles = val.listRoles;
+
+                return val.listUser
+            }
+        },
+        columns: [
+            {
+                class: "check-row text-center",
+                sortable: false,
+                data: function (val) {
+                    if (currentId != val.user.Id) {
+                        return "<input type='checkbox' name='check-select-user-for-action' data-user-id='" + val.user.Id + "'>";
+                    }
+                }
+            },
+            {
+                data: "user.LivrETSID",
+                visible: false
+            },
+            {
+                data: "user.FirstName"
+            },
+            {
+                data: "user.LastName"
+            },
+            {
+                data: "user.BarCode"
+            },
+            {
+                data: function (val) {
+                    return new Date(parseInt(val.user.SubscribedAt.replace('/Date(', ''))).toDateString();
+                }
+            },
+            {
+                data: function (val) {
+                    var html = "<div class='dropdown'>"
+                    html += "<button data-toggle='dropdown' id='" + val.user.Id + "' class='btn btn-default btn-sm dropdown-toggle' aria-haspopup='true' " +
+                            "aria-expanded='true'>" + val.role + " <span class='caret'></span></button>";
+                    html += "<ul class='dropdown-menu' aria-labelledby='" + val.user.Id + "'>";
+
+                    for (var i = 0; i < listRoles.length; i++) {
+                        if (val.role != listRoles[i].Name) {
+                            html += "<li><a href='#' data-rolename='" + listRoles[i].Name + "' data-userid='" + val.user.Id + "' class='elt-role'>" +
+                                listRoles[i].Name + "</a></li>";
+                        }
+                    }
+
+                    //html += "<li><input type='radio' id='User' name='UserRole' value='User'><label for='User'>User</label></li>";
```

Comment:
je parle du code en commentaire. 
`<li><input type='radio' id='User' name='UserRole' value='User'><label for='User'>User</label></li>`
Je ne comprends pas pourquoi il a été hard code. Les droits des users devraient tous venir de la BD